bundles/org.openhab.binding.solarforecast/README.md

```diff
@@ -0,0 +1,346 @@
+You need to configure your home photovoltaic system within the web interface.
+The `resourceId` for each PV plane is provided afterwards.
+
+In order to receive proper timestamps double check your time zone in *openHAB - Settings - Regional Settings*.
+Correct time zone is necessary to show correct forecast times in UI. 
+
+### Solcast Tuning
+
+You have the opportunity to [send your own measurements back to Solcast API](https://legacy-docs.solcast.com.au/#measurements-rooftop-site).
+This data is used internally to improve the forecast for your specific site.
+Configuration and channels can be set after checking the *Show advanced* checkbox.
+You need an item which reports the electric power for the specific rooftop. 
+If item isn't set, no measures will be sent.
+As described in [Solcast Rooftop Measurement](https://legacy-docs.solcast.com.au/#measurements-rooftop-site) check in beforehand if your measures are *sane*.
+
+- item is delivering correct values and they are stored in persistence
+- time zone setting in openHAB is correct to deliver correct timestamp
+
+After measurement is sent the `raw-tuning` channel is reporting the result.
+
+### Solcast Bridge Configuration
+
+| Name                   | Type    | Description                           | Default     | Required | Advanced |
+|------------------------|---------|---------------------------------------|-------------|----------|----------|
+| apiKey                 | text    | API Key                               | N/A         | yes      | no       |
+| timeZone               | text    | Time Zone of forecast location        | empty       | no       | yes      |
+
+`apiKey` can be obtained in your [Account Settings](https://toolkit.solcast.com.au/account)
+
+`timeZone` leave it empty to evaluate Regional Settings of your openHAB installation. 
+See [DateTime](#date-time) section for more information.
+
+### Solcast Plane Configuration
+
+| Name            | Type    | Description                                            | Default         | Required | Advanced |
+|-----------------|---------|--------------------------------------------------------|-----------------|----------|----------|
+| resourceId      | text    | Resource Id of Solcast rooftop site                    | N/A             | yes      | no       |
+| refreshInterval | integer | Forecast Refresh Interval in minutes                   | 120             | yes      | no       |
+
+`resourceId` for each plane can be obtained in your [Rooftop Sites](https://toolkit.solcast.com.au/rooftop-sites)
+
+`refreshInterval` of forecast data needs to respect the throttling of the Solcast service. 
+If you have 25 free calls per day, each plane needs 2 calls per update a refresh interval of 120 minutes will result in 24 calls per day.
+
+Note: `channelRefreshInterval` from [Bridge Configuration](#solcast-bridge-configuration) will calculate intermediate values without requesting new forecast data.
```

Comment:
This does not exist?

Reply:
Removed - channelRefreshInterval was removed after rework